src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -465,5 +475,62 @@ def _flexible_server_params(command_group):
         with self.argument_context('{} flexible-server replica stop-replication'.format(command_group)) as c:
             c.argument('server_name', options_list=['--name', '-n'], help='Name of the replica server.')
 
+    def handle_migration_parameters(command_group, server_name_arg_type, subscription_arg_type, migration_id_arg_type):
+        for scope in ['create', 'show', 'list', 'update', 'delete']:
+            argument_context_string = '{} flexible-server migration {}'.format(command_group, scope)
+            with self.argument_context(argument_context_string) as c:
+                c.argument('subscription_id', arg_type=subscription_arg_type, options_list=['--subscription-id'],
+                           help='Subscription ID of the migration target server.')
```

Comment:
In addition, I want to check with you if the subscription could be in different tenant from source one.